examples/file-and-cloudinary-upload/README.md

```diff
@@ -0,0 +1,38 @@
+# upload file and images
+
+This is a simple example of using the remix buildin [uploadHandler](https://remix.run/docs/en/v1/api/remix#uploadhandler) and Form with multipart data to upload an image file and display it,
+it also show a simple(though not efficient way) of integrate with cloudinary without writing custom handler.
+
+the relevent files are:
+
+```
+├── app
+│   ├── routes
+│   │   ├── cloudinary-upload.tsx // upload to cloudinary
+│   │   └── local-upload.tsx // local upload using build in [createfileuploadhandler](https://remix.run/docs/en/v1/api/remix#unstable_createfileuploadhandler)
+│   └── utils
+│       └── utils.server.ts  // init cloudinary nodejs client on server side
+|── .env // holds cloudinary credentails
+```
+
+## steps to set up cloudinary
+
+- sign up a free [cloudinary account](https://cloudinary.com/)
+- get the cloudname, api key and api secret from dashboard
+- copy the .env.sample to .env and populate the credentials
+
+## caveat
+
+- upload large file is not seems stable now, try upload image with size < 30kb
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+<!-- TODO: update this link to the path for your example: -->
+
+[![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/file-and-cloudinary-upload)
+
+## Example
+
+## Related Links
+
+Link to documentation or other related examples.
```

Comment:
@marvinwu could you add links to documentation here please?

Reply:
@kentcdodds thx much for reviewing! sure will update it with doc links

Reply:
@kentcdodds fixed, kindly take a look 

[updated readme](https://github.com/remix-run/remix/commit/072560795c54b229f2cad52776e16b09af10283e#diff-38bd30ea75140e0835f0538470e537a5be357755a5719294833d20bc10a748f0)